### PR TITLE
Provide ENV variables support

### LINKED ISSUE #35: Provide ENV variables support
- pass some vars into project
- use new approach to retrieve and pass vars into services

### PR DESCRIPTION
**Pull request by nightael**
- [x] pass some vars into project
- [x] use new approach to retrieve and pass vars into services

Closes #35

### CODE REVIEW

File: src/com/apmath/loans/infrastructure/Module.kt [hunk 1]
```diff
@@ -14,9 +15,27 @@ val loans = module {
     //services
```
Comment (BarracudaPff) [inline src/com/apmath/loans/infrastructure/Module.kt:17]: add @KtorExperimentalAPI

Comment (BarracudaPff) [reply src/com/apmath/loans/infrastructure/Module.kt:17]: over module

File: src/com/apmath/loans/infrastructure/fetchers/AbstractFetcher.kt [hunk 2]
```diff
@@ -13,14 +13,13 @@ import io.ktor.http.contentType
...
-) {
+open class AbstractFetcher(host: String, port: Int) {
+    val host: String = host
     suspend inline fun <reified T> get(path: String): T {
         return AbstractFetcher.client.request {
             method = HttpMethod.Get
             url(
-                host = host.value,
+                host = host,
                 path = path
```
Comment (BarracudaPff) [inline src/com/apmath/loans/infrastructure/fetchers/AbstractFetcher.kt:23]: also, add port in all methods

**Review by BarracudaPff — CHANGES_REQUESTED** [review]

File: src/com/apmath/loans/infrastructure/fetchers/AbstractFetcher.kt [hunk 3]
```diff
@@ -13,14 +13,15 @@ import io.ktor.http.contentType
 import io.ktor.http.isSuccess
 
 //TODO: temporary implementation
-open class AbstractFetcher(
-    val host: Host
-) {
+open class AbstractFetcher(host: String, port: Int) {
+    val host: String = host
```
Comment (BarracudaPff) [inline src/com/apmath/loans/infrastructure/fetchers/AbstractFetcher.kt]: Перенеси val прямо в конструктор для `host` и `port`

**Review by BarracudaPff — CHANGES_REQUESTED** [review]

**Review by BarracudaPff — APPROVED** [review]

**Review by malinink — APPROVED** [review]

**Pull request merged**